casadm/cas_lib.h

```diff
@@ -298,6 +298,11 @@ void print_err(int error_code);
   */
 int get_dev_path(const char* disk, char* buf, size_t num);
 
+/**
+  * @brief get special device by-id link for disk.
```

Comment:
I would be more specific here. Assuming you decide to return sanitized path from this function (abs_dev_path) then this would make sense:

@brief make sure device path is unique and write sanitized version to @path

Reply:
done